subworkflows/local/samplesheet_to_channel/main.nf

```diff
@@ -0,0 +1,60 @@
+workflow SAMPLESHEET_TO_CHANNEL {
+    take:
+    reference
+
+    main:
+
+    intervals_bed = reference.map { meta, input_intervals_bed, input_fasta, input_fasta_dict, input_fasta_fai, input_fasta_sizes, input_gff, input_gtf, input_splice_sites, input_transcript_fasta, input_vcf, input_readme, input_bed12, input_mito_name, input_macs_gsize ->
```

Comment:
wouldn't a multimap save the repetition of the map operation?

Reply:
Maybe output a string 'NULL' or something and then have a map to switch it out to a NULL in the emits?

Imagine having to fix all of these if the input channel changes in structure.

Reply:
that's what I had before, and I had to remap every single channel, so I'd rather just use multiple maps for now rather than multiMap + multiple maps.
I'll make an MRE for Ben and others to solve